src/struct_mv/struct_matmult.c

```diff
@@ -13,6 +13,7 @@
 
 #include "_hypre_struct_mv.h"
 #include "_hypre_struct_mv.hpp"
+//#define USE_FUSED_MATMULT 1
```

Comment:
The results you showed in our last meeting showed that the fused version was a bit faster, yes? At least on the GPU? Should we enable this by default, or is there a reason not to?

Reply:
I think we should enable it by default. I left it disabled to make just to my tests easier :)